content/en/blog/2024/otel-errors/index.md

```diff
@@ -0,0 +1,286 @@
+for throwing and catching exceptions. 
+
+When you begin at a place where different languages disagree about what an error 
+or exception is and how to handle them, where do you go when you need standardized 
+telemetry and error reporting across microservices written in those languages? 
+OpenTelemetry is the key to this answer, and is the lens through which we’ll 
+address the following, and more:
+
+* How an error is visualized in a backend may not be where you think it’ll be, 
+or how you expect it to look. 
+* How span kind affects error reporting. 
+* Errors reported by spans vs logs.
+
+## Errors vs Exceptions
+Before we get into how OTel deals with errors and exceptions, let’s establish 
+what they are, and how they’re different from each other. While there are 
+variations on the definitions of these terms, we’ve landed on the ones below, 
+which we’ll be using in this article:
+
+An **error** is an unexpected issue in a program that hinders its execution. 
+Examples include syntax errors, such as a missing semicolon or incorrect 
+indentation, and runtime errors, resulting from errors in logic.
+
+An **exception** is a type of runtime error that disrupts the normal flow of a 
+program. Examples include dividing by zero or accessing an invalid memory address. 
+
+Some languages, such as Python and JavaScript, treat errors and exceptions as 
+synonyms; others, such as PHP and Java, do not. Understanding the distinction 
+between the two is crucial for effective error handling. By recognizing the 
+differences between errors and exceptions, you can adopt more nuanced strategies 
+for handling and recovering from failures in your applications.
+
+## Errors in OpenTelemetry
+So how does OTel deal with all these conceptual differences across languages? 
+This is where the [specification](https://opentelemetry.io/docs/specs/otel/) (or 
+“spec” for short) comes in. The spec provides a blueprint for developers working 
+on various parts of the project, and standardizes implementation across all 
+languages. 
+
+Since language APIs and SDKs are implementations of the spec, there are general 
+rules against implementing anything that isn’t covered in the spec. This provides 
+a guiding principle to help organize contributions to the project. In practice, 
+there are a few exceptions; for example, a language might prototype a new feature 
+as part of adding it to the spec, but the feature may be published (usually as 
+alpha or experimental) before the corresponding language is added. 
+
+Another exception is when a language might decide to diverge from the spec. 
+Although it is generally not advised, sometimes there are strong language-specific 
+reasons to do something different. In this way, the spec allows for some 
+flexibility for each language to implement features as idiomatically as possible. 
+For example, most languages have implemented `RecordException`, while Go has 
+implemented `RecordError`, which does the same thing. 
```

Comment:
does not have to be linked in the blog post, but can you share some evidence for this being true (not that go uses a different name, but that the spec allows that flexibility)

Reply:
no, I think that's good enough, I was assuming this is in the spec somewhere.